server/datastore/mysql/mysql.go

```diff
@@ -62,6 +62,22 @@ type Datastore struct {
 
 type txFn func(sqlx.ExtContext) error
 
+type entity struct {
+	name string
+}
+
+var (
+	hostsTable            = entity{"hosts"}
+	invitesTable          = entity{"invites"}
+	labelsTable           = entity{"labels"}
+	packsTable            = entity{"packs"}
+	queriesTable          = entity{"queries"}
+	scheduledQueriesTable = entity{"scheduled_queries"}
+	sessionsTable         = entity{"sessions"}
+	teamsTable            = entity{"teams"}
+	usersTable            = entity{"users"}
+)
```

Comment:
We can probably go generate these from the sql schema. But we can do that iteration later.